src/shared/type-mapper.ts

```diff
@@ -57,6 +65,18 @@ export abstract class TypeMapperImpl implements TypeMapper, Importer {
             case KnownTypes.Reference: {
                 return ((node as TypeReferenceNode).typeName as Identifier).escapedText!
             }
+            case KnownTypes.Array: {
+                let typeKind, knownTypeKind;
+                let knownType = this.getKnownType(kind);
+                if (this.isGenericArray(node)) {
+                    typeKind = this.toKnownType(((node as NodeWithTypeArguments).typeArguments as NodeArray<TypeNode>)[0]);
+                    knownTypeKind = this.getKnownType(typeKind);
```

Comment:
dupe of 76, place it out of the if else block?

Reply:
Done!